examples/agents/bee_granite.ts

```diff
@@ -0,0 +1,72 @@
+import "dotenv/config.js";
+import { FrameworkError } from "bee-agent-framework/errors";
+import { TokenMemory } from "bee-agent-framework/memory/tokenMemory";
+import { DuckDuckGoSearchTool } from "bee-agent-framework/tools/search/duckDuckGoSearch";
+
+import { createConsoleReader } from "examples/helpers/io.js";
+import { GraniteBeeAgent } from "bee-agent-framework/agents/granite/agent";
+import { WatsonXChatLLM } from "bee-agent-framework/adapters/watsonx/chat";
+import { z } from "zod";
+import { parseEnv } from "bee-agent-framework/internals/env";
+
+// Ensure that you have added your WatsonX credentials in .env
+const llm = WatsonXChatLLM.fromPreset("ibm/granite-3-8b-instruct", {
+  apiKey: parseEnv("WATSONX_API_KEY", z.string()),
+  projectId: parseEnv("WATSONX_PROJECT_ID", z.string()),
+  parameters: {
+    decoding_method: "greedy",
+    max_new_tokens: 1024,
+  },
```

Comment:
You are missing a stop sequence.

Reply:
I moved the max_new_tokens into granite8b preset for watsonx [src/adapters/watsonx/chatPreset.ts](https://github.com/i-am-bee/bee-agent-framework/pull/159/files#diff-aadf764b9e5cf6016898f56145c827099fa6cc300339667531a2eb4cc6daee65) so I no longer need to configure the parameters in the example.